### PR TITLE
[FIX] Fix Twitter card

### PR DESCRIPTION
**Pull request by csouchet**
- https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/summary
- https://cards-dev.twitter.com/validator

### CODE REVIEW

**Comment by csouchet** [comment]
### At commit 0214cca
image

File: src/theme/components/Helmet.tsx [hunk 1]
```diff
@@ -13,8 +14,16 @@ type Props = {
 const LANG = 'en';
 
 const Helmet = ({ theme, title, keywords }: Props): JSX.Element => {
-  const { description, profile } = helmet;
-
+  const { description, logo, twitter } = helmet;
+  const data = useStaticQuery(graphql`
```
Comment (tbouffard) [inline src/theme/components/Helmet.tsx:18]: nice 👍🏿

File: src/content/HelmetContent.tsx [hunk 2]
```diff
@@ -1,9 +1,11 @@
-import SvgLogoWhite from '../images/logo_white.svg';
 import { landing } from './LandingContent';
 
 export const helmet = {
   description: landing.presentation, // This should not be too long (160 chars)  see https://www.gatsbyjs.com/tutorial/seo-and-social-sharing-cards-tutorial/#querying-with-graphql
-  profile: {
-    logo: { src: SvgLogoWhite },
+  logo: {
+    src: '/image/twitter_card.webp',
```
Comment (tbouffard) [inline src/content/HelmetContent.tsx:6]: ⚠️ property name is not accurate: this is for usage in `og:image`
Same remarks for the image file name

File: src/theme/components/Footer.tsx [hunk 3]
```diff
@@ -10,7 +10,7 @@ import { helmet } from '../../content/HelmetContent';
 
 const Footer = (): JSX.Element => {
   const { title, socialLinks } = landing;
-  const { profile } = helmet;
+  const { logo } = helmet;
```
Comment (tbouffard) [inline src/theme/components/Footer.tsx:13]: ⚠️ The footer now displays the image created for og:image
We should restore the usage of the white logo that is also used in the header.

image

**Review by tbouffard — CHANGES_REQUESTED** [review]

File: src/content/HelmetContent.tsx (hunk 2 above)
Comment (csouchet) [reply src/content/HelmetContent.tsx]: Done

File: src/theme/components/Footer.tsx (hunk 3 above)
Comment (csouchet) [reply src/theme/components/Footer.tsx]: Fixed

**Review by tbouffard — APPROVED** [review]

**Pull request merged**